src/ScriptBuilder/Outbox/Create_PostgreSql.sql

```diff
@@ -24,4 +24,4 @@
   $body$
   language 'plpgsql';
 
-select create_outbox_table(@tablePrefix);
\ No newline at end of file
+select pg_temp.create_outbox_table(@tablePrefix);
```

Comment:
> pg_temp.

Making a mental note, we should document using that I think. I'll research more to see if there are any potential problems with this approach. E.g. what if the connection is dropped in the middle of script creation?